assets/sass/components/_frontpage-cards.scss

```diff
@@ -0,0 +1,129 @@
+
+.frontpage {
+  &.frontpage-section {
+    margin: 1%;
```

Comment:
These use indentation of 2 spaces, while rest of the file tabs (set in .editorconfig).

Reply:
Thanks, I'd not got that far in getting up-to-speed with the changes. I think I updatd this now.